### PR TITLE
feat(FOROME-1554): redesign condition panel

### PR DESCRIPTION
**Pull request by EvgeniySobolev**
[https://quantori.atlassian.net/browse/FOROME-1554](https://quantori.atlassian.net/browse/FOROME-1554)

### CODE REVIEW

File: src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx [hunk 1]
```diff
@@ -62,47 +81,57 @@ export const ConditionCard = observer(
...
-        )}
-      >
-        <div className={styles.conditionCard__title}>
+      <>
+        <div
+          className={cn(
+            styles.conditionCard,
+            isActive && styles.conditionCard_active,
+            className,
+            'mx-4 my-2',
```
Comment (QSdmitrioul) [inline src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx]: move it to css module

File: src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx [hunk 2]
```diff
@@ -62,47 +81,57 @@ export const ConditionCard = observer(
...
-            onClick={() => setContentVisible(!isContentVisible)}
           >
-            <Icon name="Arrow" />
-          </button>
-          <ConditionOptions
-            className={styles.conditionCard__options}
-            filterName={filterName}
-            onDelete={onDelete}
-          />
+            <div className={cn(styles.conditionCard__filterName, 'mr-1')}>
```
Comment (QSdmitrioul) [inline src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx]: move mr-1 to css module

File: src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx [hunk 3]
```diff
@@ -62,47 +81,57 @@ export const ConditionCard = observer(
...
+                  isContentVisible && 'mt-2',
+                )}
+              >
+                <ConditionContent
+                  isPreview={isPreviewMode}
+                  condition={condition}
+                />
+              </div>
+            )}
+            {isPreviewMode && !isContentHidden && (
```
Comment (QSdmitrioul) [inline src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx]: extract to variable

**Comment by Lapk1n** [comment]
1. Condition container border-radius must be 8px
2. Increase margins between condition filters 
Screenshot 2022-08-24 at 17 47 59

1. Does not math to market (0 < ... <= 2)
2. All these blocks have dnd inside but there is no cursor grab on hover
3. If I use dnd for some folded container it unfolds, that's annoying behaviour.
Screenshot 2022-08-24 at 18 11 33

1. Increase margins between sections (Inheritance_Mode / Problem group / Inheritance type)
Screenshot 2022-08-24 at 18 23 53

1. As close as atoms)
Screenshot 2022-08-24 at 18 27 34

File: src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx [hunk 4]
```diff
@@ -39,7 +37,28 @@ export const ConditionCard = observer(
         ? (condition[2] as ConditionJoinMode)
         : undefined
 
-    const [isContentVisible, setContentVisible] = useState(true)
+    const isArrowShown = useMemo(() => {
+      if (filterType === AttributeKinds.FUNC) {
+        return true
+      }
+      if (filterType !== AttributeKinds.ENUM) {
```
Comment (Lapk1n) [inline src/pages/filter/refiner/components/right-column/condition-card/condition-card.tsx]: No need for this checking. Numeric condition length are always less than 1

**Comment by Lapk1n** [comment]
If we apply some preset we update the URL, so clicking on the clear all button must refresh the URL too

**Review by Lapk1n — COMMENTED** [review]
Screenshot 2022-08-30 at 17 41 25
The compound request panel got some changes (Scenario => At least counts)

**Review by Lapk1n — APPROVED** [review]

File: src/pages/filter/refiner/components/right-column/filter-right-column.module.css [hunk 5]
```diff
@@ -16,4 +16,21 @@
     flex: 1 0 0;
     overflow: auto;
   }
+
+  &__action {
+    font-weight: 500;
+    margin: 0 0 0 8px;
```
Comment (QSdmitrioul) [inline src/pages/filter/refiner/components/right-column/filter-right-column.module.css]: margin-left: 8px???

**Review by QSdmitrioul — APPROVED** [review]

**Pull request merged**